src/libs/DateUtils.ts

```diff
@@ -730,6 +730,26 @@ function formatToSupportedTimezone(timezoneInput: Timezone): Timezone {
     };
 }
 
+/**
+ * Returns the latest business day of input date month
+ *
+ * param {Date} inputDate
+ * returns {number}
+ */
+function getLastBusinessDayOfMonth(inputDate: Date): number {
+    const currentDate = new Date(inputDate);
+
+    // Set the date to the last day of the month
+    currentDate.setMonth(currentDate.getMonth() + 1, 0);
+
+    // Loop backward to find the latest business day
+    while (currentDate.getDay() === 0 || currentDate.getDay() === 6) {
```

Comment:
We can move currentDate.getDay() to const dayOfWeek = getDay(currentDate);
Optional:
Do we need to loop through the dates?
We can simplify make it with if statement: 
if(dayOfWeek === 0) currentDate = subDays(currentDate, 2)
else if(dayOfWeek === 6) currentDate = subDays(currentDate, 1)
else currentDate

Reply:
Thank you so much. It works. Done - 453de73.